Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'wicked_pdf'
 gem 'wkhtmltopdf-binary'
 gem 'activeadmin_addons'
 gem 'bootstrap-datepicker-rails'
+gem 'regexp-examples', '~> 1.5', '>= 1.5.1'
```

Comment:
This gem converts a regular expression to some strings which match that regular expression. I thought it'd be helpful to show the administrator what their regex does.

![Screenshot_20221110_054034](https://user-images.githubusercontent.com/11529449/200914397-c698a9a4-b955-4e06-b7b4-0b665960625a.png)

Reply:
Neat! Can it be used as generator for tests?

Reply:
Property-based tests are cool but I don't see a way to use them here. Do you have a concrete idea for how I could test?